documentation/modules/ROOT/pages/crafting-deployments-iac.adoc

```diff
@@ -250,6 +257,26 @@ spec:
 EOF
 -----
 
+It is strongly recommended to include crun manifests as part of the additional install-time manifests for 4.13+. So let's create proper machine configuration in the extra manifests folder:
+
+[.console-input]
+[source,bash,subs="attributes+,+macros"]
+-----
+cat <<EOF > ~/5g-deployment-lab/ztp-repository/site-configs/hub-1/sno2-extra-manifest/enable-crun-master.yaml
```

Comment:
Shouldn't we add the same for workers? I know this is an SNO, but if in the future the SNO gets expanded to SNO+Workers these wouldn't have crun, right?

Reply:
Included worker machineconfig for crun too.